modules/set-the-default-max-container-root-partition-size-for-overlay-with-crio.adoc

```diff
@@ -0,0 +1,122 @@
+
+The root partition of each container shows all of the available disk space of the underlying host. Follow this guidance to set a maximum partition size for the root disk of all containers.
+
+To configure the maximum Overlay size, as well as other CRI-O options like the log level and PID limit, you can create the following `ContainerRuntimeConfig` CRD:
+
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: overlay-size
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     custom-crio: overlay-size
+ containerRuntimeConfig:
+   pidsLimit: 2048
+   logLevel: debug
+   overlaySize: 8G
+----
+
+.Procedure
+
+. Create the configuration object:
++
+[source,terminal]
+----
+$ oc apply -f overlaysize.yml
+----
+
+. To apply the new CRI-O configuration to your worker nodes, edit the worker machine config pool:
++
+[source,terminal]
+----
+$ oc edit machineconfigpool worker
+----
+
+. Add the `custom-crio` label based on the `matchLabels` name you set in the `ContainerRuntimeConfig` CRD:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  creationTimestamp: "2020-07-09T15:46:34Z"
+  generation: 3
+  labels:
+    custom-crio: overlay-size
+    machineconfiguration.openshift.io/mco-built-in: ""
+----
+
+. Once saved, view the machine configs:
++
+[source,terminal]
+----
+$ oc get machineconfigs
+----
++
+You will see a new `99-worker-generated-containerruntime` object as well as a new `rendered-worker-xyz` object created:
++
+.Example output
+[source,terminal]
+----
+99-worker-generated-containerruntime  4173030d89fbf4a7a0976d1665491a4d9a6e54f1   2.2.0             7m42s
+rendered-worker-xyz                   4173030d89fbf4a7a0976d1665491a4d9a6e54f1   2.2.0             7m36s
+----
+
+. Once those objects are created, monitor the machine config pool for the changes to be applied:
++
+[source,terminal]
+----
+$ oc get mcp worker
+----
++
+The worker nodes show `UPDATING` as `True` with the number of machines, the number updated, and other details:
++
+.Example output
+[source,terminal]
+----
+NAME   CONFIG              UPDATED   UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+worker rendered-worker-xyz False True False     3             2                   2                    0                      20h
+----
++
+When complete, the worker nodes will transition back to `UPDATING` as `False` with the number of `UPDATED` matching the `MACHINECOUNT`:
++
+.Example output
+[source,terminal]
+----
+NAME   CONFIG              UPDATED   UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+worker   rendered-worker-xyz   True      False      False      3         3            3             0           20h
+----
++
+Looking at a worker machine, you see the new 8GB max size configuration is applied to all of the workers:
```

Comment:
Small nit, and for localization best practice, you could add "that" here. For example:

> Looking at a worker machine, you see that the new 8 GB max size configuration is applied to all of the workers:

Looks like ISG wants a space before GB as well.